external-builds/pytorch/build_prod_wheels.py

```diff
@@ -174,7 +174,7 @@ def directory_if_exists(dir: Path) -> Path | None:
 def do_install_rocm(args: argparse.Namespace):
     # Because the rocm-sdk package caches current GPU selection and such, we
     # always purge it to ensure a clean rebuild.
-    exec([sys.executable, "-m", "pip", "cache", "remove", "rocm_sdk"], cwd=Path.cwd())
+    # exec([sys.executable, "-m", "pip", "cache", "remove", "rocm_sdk"], cwd=Path.cwd())
```

Comment:
Revert the change to this line (when requesting review, please keep debug changes off of the pull request so we can focus on the actual changes)

Reply:
I got the same error: https://github.com/ROCm/TheRock/actions/runs/15352833360/job/43205051969

Reply:
Removing for now to keep my testing going.  We can talk about it after the other things are done.

Reply:
You may need to debug. You should be able to reproduce that locally inside or outside of Docker, separate from github actions, forks, etc.

Reply:
This is also looks like permission issue
I might add this below:
```
try:
    exec([sys.executable, "-m", "pip", "cache", "remove", "rocm_sdk"], cwd=Path.cwd())
except subprocess.CalledProcessError:
    print("[WARN] Skipping 'pip cache remove rocm_sdk' due to cache being unavailable.")
```
@stellaraccident could you please advice?

Reply:
Synced online with Stella offline and we will use the code as is since the workflow running in a docker container. Will use
--pip-cache-dir /tmp/pipcache option to avoid permission problem.